### PR TITLE
Upgrade/500 600/logical enclosure

### PR DESCRIPTION
**Pull request by sijeesh**
Logical enclosure ::Added support for API version 500 and 600

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass (`$ rake test`).
- [x] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

**Review by jyothisgm — APPROVED** [review]
LGTM

File: examples/logical_enclosure.pp [hunk 1]
```diff
@@ -25,35 +25,43 @@
 # }
 
 oneview_logical_enclosure{'logical_enc1':
-    ensure => 'get_script',
+    ensure => 'reapplay_configuration',
```
Comment (soodpr) [inline examples/logical_enclosure.pp]: can you rename it as reapply_configuration?

**Review by madhav-bharadwaj — APPROVED** [review]
Looks good to me.

**Review by soodpr — APPROVED** [review]
LGTM 👍

**Pull request merged**